.github/workflows/lint.yml

```diff
@@ -33,17 +33,3 @@ jobs:
         uses: pre-commit/action@v3.0.1
         with:
           extra_args: --all-files --hook-stage manual codespell || true
-
-  check-peps:
-    name: Run check-peps
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v4
-      - name: Set up Python 3
-        uses: actions/setup-python@v5
-        with:
-          python-version: "3"
-
-      - name: Run check-peps
-        run: python check-peps.py --detailed
```

Comment:
Any reason to remove this? It currently finishes before `pre-commit` has fully initialised itself!

Reply:
We don't need to run it twice, and have it potentially fail in two places. We need to wait for pre-commit to finish initialising anyway.